styles.css

```diff
@@ -7,6 +7,10 @@ body{
   flex-wrap: wrap;
   font-family
 }
+body> * {
+  margin:0;
+}
+
```

Comment:
Getting there. We want to apply `margin: 0` to the direct children of `header` `footer` `aside` `main`, not `body`. So your answer should look something like this, using `header` as an example...

```css
header > * {
  margin: 0;
}
```